scripts/get-keptn-payload.js

```diff
@@ -0,0 +1,25 @@
+module.exports = ({ context }) => {
+  const labels = context.payload.pull_request.labels;
+  const parsedLabels = {};
+  labels.forEach((label) => {
+    const [key, value] = label.name.split(":");
+    parsedLabels[key] = value;
+  });
+
+  return JSON.stringify({
+    data: {
+      project: parsedLabels["project"],
+      stage: parsedLabels["stage"],
+      service: parsedLabels["service"],
+      // What should we send if the PR is closed without merging?
```

Comment:
I wasn't sure what to do if the PR is closed. We should tell Keptn but should the result be "fail"?

Reply:
I agree we should tell Keptn and return a `result: "pass"`. Then Keptn will continue and release what is in the repo - in this case, the not-updated content. From a story point of view, this should be okay.